docs/src/pages/styles/typography/typography-body.example.html

```diff
@@ -0,0 +1,14 @@
+<section class="example--list">
```

Comment:
This file doesn't need to be added - it's identical to the core

Reply:
Will do, I'll remove it.